src/InstagramClone.Api/Database/InstagramCloneDbContext.cs

```diff
@@ -29,30 +29,49 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<PostReaction>().HasOne<Post>(reaction => reaction.Post)
                 .WithMany(posts => posts.PostReactions)
                 .HasForeignKey(fk => fk.PostId);
+
+
+
+            modelBuilder.Entity<UserFollower>()
+                .HasOne(following => following.Follower)
+                .WithMany(followers => followers.Users);
+
+            modelBuilder.Entity<UserFollower>()
+                .HasOne(f => f.Follower)
+                .WithMany(u => u.Followed);
+
+            modelBuilder.Entity<UserFollower>()
+                .HasOne(fl => fl.FollowedUser)
+                .WithMany( w => w.Followers);
```

Comment:
`fl.followedUser` should be configured to match `u.Followed` and voice versa. I hope u get

Reply:
Yes I do

Reply:
so yes, just fix it right away